stores/multipart.go

```diff
@@ -274,7 +275,7 @@ func (s *SQLStore) CompleteMultipartUpload(ctx context.Context, bucket, path str
 				} else if mu.Parts[j].PartNumber > part.PartNumber {
 					// missing part
 					return api.ErrPartNotFound
-				} else if mu.Parts[j].PartNumber == part.PartNumber && mu.Parts[j].Etag == part.ETag {
+				} else if mu.Parts[j].PartNumber == part.PartNumber && mu.Parts[j].Etag == strings.Trim(part.ETag, "\"") {
```

Comment:
I think this could've been stripped properly in `backend.go`? 
Should we add some type of etag sanitizer?